container_pipeline/lib/default_settings.py

```diff
@@ -0,0 +1,68 @@
+LOG_LEVEL = 'DEBUG'
+
+LOG_PATH = '/srv/pipeline-logs/cccp.log'
+
+LOGGING_CONF = dict(
+    version=1,
+    level=LOG_LEVEL,
+    formatters=dict(
+        bare={
+            "format": ('[%(asctime)s] %(name)s p%(process)s %(lineno)d '
+                       '%(levelname)s - %(message)s')
+        },
+    ),
+    handlers=dict(
+        console={
+            "class": "logging.StreamHandler",
+            "formatter": "bare",
+            "level": LOG_LEVEL,
+        },
+        log_to_file={
+            'level': 'DEBUG',
+            'class': 'logging.FileHandler',
+            'filename': LOG_PATH,
+            'mode': 'a+',
+            'formatter': 'bare',
+        }
+    ),
+    loggers={
+        'linter': {
+            "level": "DEBUG",
+            "propagate": False,
+            "handlers": ["console", "log_to_file"],
+        },
+        'build-worker': {
+            "level": "DEBUG",
+            "propagate": False,
+            "handlers": ["console", "log_to_file"],
+        },
+        'delivery-worker': {
+            "level": "DEBUG",
+            "propagate": False,
+            "handlers": ["console", "log_to_file"],
+        },
+        'scan-worker': {
+            "level": "DEBUG",
+            "propagate": False,
+            "handlers": ["console", "log_to_file"],
+        },
+        'dispatcher': {
+            "level": "DEBUG",
+            "propagate": False,
+            "handlers": ["console", "log_to_file"],
+        },
+        'mail-service': {
+            "level": "DEBUG",
+            "propagate": False,
+            "handlers": ["console", "log_to_file"],
+        },
+        'console': {
+            "level": "DEBUG",
+            "propagate": False,
+            "handlers": ["console"]
+        }
```

Comment:
We aren't using var ``LOG_LEVEL`` here because we wan't the loggers to generate all logs, and let the handlers decide which log levels to capture.